oms-wiki/docker/LocalSettings.php.example

```diff
@@ -30,33 +30,33 @@ $wgMetaNamespace = "AEGEE_wiki";
 $wgScriptPath = "";
 
 ## The protocol and server name to use in fully-qualified URLs
-$wgServer = "https://wiki.staging.barnab.eu";
+$wgServer = "https://".$_ENV["SUBDOMAIN"].$_ENV["URL"];
```

Comment:
Why not BASE_URL?

Reply:
i changed the var names inside

Reply:
reasonable. I'd rather leave it as SUBDOMAIN_WIKI and BASE_URL, but that's not that important, it'll work that way as well.

Reply:
fine